hugegraph-server/hugegraph-rocksdb/src/main/java/org/apache/hugegraph/backend/store/rocksdb/RocksDBStore.java

```diff
@@ -414,6 +414,16 @@ public void close() {
         this.closeSessions();
     }
 
+    public void forceClose() {
+        try {
+            this.checkOpened();
+            this.closeSessions();
+        } catch (Throwable ignore) {
+            return;
+        }
+        ((RocksDBStdSessions)this.sessions).forceClose();
```

Comment:
may the rocksdb instance be shared by other graph instance

Reply:
now, if  the rocksdb instance  be shard with other graph instance, the rocksdb will throw exception, now, different graph have different rocksdb instance